### PR TITLE
ImageProcessing _image_rotation

### PR DESCRIPTION
**Pull request by ryuk156**
issue : https://github.com/ankitdobhal/Awesome-Python-Scripts/issues/93

About Pull request: In this Pull request I have added script which shows how to rotate an image using openCV

Demo : 
Non-rotated
image

rotated image:

image

### CODE REVIEW

**Comment by ryuk156** [comment]
@disha2sinha review please

File: Image-Processing/image _rotation.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import cv2 ##import the module
+img = cv2.imread("rose.jpg") ##read your image 
+
+cv2.imshow('Original Image', img)  ##this will display the image original image
+cv2.waitKey(0)
+
```
Comment (disha2sinha) [inline Image-Processing/image _rotation.py]: Remove unnecessary new lines

File: Image-Processing/image _rotation.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+##.shape will give you height and width of image
+height, width = img.shape[0:2]
+##getrotationmatrix2d will create a rotation matrix  "(center, angle, scale)" => arguments
+rotationMatrix = cv2.getRotationMatrix2D((width/2, height/2), 90, .5)
+## the wrapAffine function will rotate image "(img, rotationMatrix, (width, height))" => arguments
+rotatedImage = cv2.warpAffine(img, rotationMatrix, (width, height)) 
+
```
Comment (disha2sinha) [inline Image-Processing/image _rotation.py]: Remove new line

File: Image-Processing/image _rotation.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+cv2.imshow('Rotated Image', rotatedImage)
+
```
Comment (disha2sinha) [inline Image-Processing/image _rotation.py]: remove new lines

**Review by disha2sinha — CHANGES_REQUESTED** [review]

**Comment by ryuk156** [comment]
@disha2sinha  done review, please

**Review by disha2sinha — APPROVED** [review]

**Pull request closed without merging**